classes/core/PKPRequest.inc.php

```diff
@@ -542,6 +542,14 @@ public function isBot()
         return $this->_isBot;
     }
 
+    /**
+     * Check if the HTTP_DNT (Do Not Track) is set
+     */
+    public function isDNTSet(): bool
```

Comment:
I would suggest renaming this function to `getDoNotTrack` -- I didn't know at a glance what DNT was until I came here.